app/viewmodels/FileStatusViewModel.scala

```diff
@@ -84,4 +84,19 @@ object FileStatusViewModel {
     Table(rows = tableRow, head = header, caption = Some(Messages("fileStatus.heading")), captionClasses = "govuk-table__caption govuk-visually-hidden")
   }
 
+  def createFileSummary(fileName: String, fileStatus: String)(implicit messages: Messages): Seq[SummaryListRow] = {
```

Comment:
@soljeggers Please place it in a separate file or as a private method in the respective controller